### PR TITLE
New package: wlrctl-0.2.1

### PR DESCRIPTION
**Pull request by ajshell1**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by ajshell1** [comment]
I've come to the realization that wrksrc should not be hardcoded, it should be "${pkgname}-v${version}". I'm attempting to fix this.

File: srcpkgs/wlrctl/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'wlrctl'
+pkgname=wlrctl
+version=0.2.1
+revision=1
+wrksrc="wlrctl-v0.2.1"
```
Comment (tibequadorian) [inline srcpkgs/wlrctl/template]: you didn't save your changes :smile:

**Review by tibequadorian — CHANGES_REQUESTED** [review]
Thanks for the credit but you don't have to mention your changes in the commit. When amending your commit nobody will see the previous version of it.
I think you can report that build error on ARM to upstream.
Also add a newline after the checksum.

File: srcpkgs/wlrctl/template (hunk 1 above)
Comment (ajshell1) [reply srcpkgs/wlrctl/template]: Fixed.

Also, please let me know if there's a policy against people using a noreply.github.com email in the repos.

Comment (tibequadorian) [reply srcpkgs/wlrctl/template]: >policy against people using a noreply.github.com email in the repos
yes: https://github.com/leahneukirchen/xtools/blob/master/xlint#L319

Comment (ajshell1) [reply srcpkgs/wlrctl/template]: Fine. I've added an email of mine.

File: srcpkgs/wlrctl/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+build_style=meson
+hostmakedepends="pkg-config scdoc wayland-devel"
+makedepends="wayland-devel libxkbcommon-devel"
+short_desc="Utility for miscellaneous wlroots extensions"
+maintainer="ajshell1 <ajshell1@protonmail.com>"
+license="MIT"
+homepage="https://git.sr.ht/~brocellous/wlrctl"
+distfiles="https://git.sr.ht/~brocellous/wlrctl/archive/v${version}.tar.gz"
+
+checksum=de8848de0a81749fe7ee7316bead0261bd978e26c2f899d02897609972a6e98c
```
Comment (ericonr) [inline srcpkgs/wlrctl/template:15]:
```suggestion
distfiles="https://git.sr.ht/~brocellous/wlrctl/archive/v${version}.tar.gz"
checksum=de8848de0a81749fe7ee7316bead0261bd978e26c2f899d02897609972a6e98c

```

**Review by ericonr — COMMENTED** [review]
The template looks fine, but I would prefer waiting for upstream to fix their bug.

**Pull request closed without merging**